Framework/CMakeLists.txt

```diff
@@ -39,16 +35,16 @@ set(
 
 set(
   SRCS_TOBJECT2JSON
-  src/TObject2Json.cxx
-  src/TObject2JsonServer.cxx
-  src/TObject2JsonWorker.cxx
-  src/TObject2JsonBackendFactory.cxx
-  src/TObject2JsonCcdb.cxx
+  src/tobject2json/TObject2Json.cxx
```

Comment:
It seems like this file was lost in the process of moving

Reply:
For me it is there. Do you see it missing ?

Reply:
ok, fixed, problem with the gitignore